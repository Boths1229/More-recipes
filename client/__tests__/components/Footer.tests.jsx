import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../../components/common/Footer';

Enzyme.configure({
  adapter: new Adapter()
});


describe('Footer component', () => {
  it('matches snapshot', () => {
    const output = shallow(<Footer />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
