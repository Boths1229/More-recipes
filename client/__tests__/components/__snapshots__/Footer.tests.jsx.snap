// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<div
  className="mt-5"
>
  <footer
    className="footer text-center"
  >
    <strong
      className="copyright"
    >
      More-Recipes Â© 2017, All Rights Reserved, Design & Developed By: Sanni Kehinde
    </strong>
  </footer>
</div>
`;
